Classes/Service/NodeRedirectService.php

```diff
@@ -199,51 +199,57 @@ protected function getLiveNode(NodeInterface $publishedNode): ?NodeInterface
     }
 
     /**
-     * Check if the current node type is disable by Settings
+     * Check if the current node type is restricted by Settings
      *
      * @param NodeInterface $publishedNode
      * @return bool
      */
-    protected function isNodeTypeDisabled(NodeInterface $publishedNode)
+    protected function isRestrictedByNodeType(NodeInterface $publishedNode)
     {
-        foreach ($this->nodeTypeConfiguration as $disabledNodeType => $status) {
-            if ($status === true) {
-                continue;
-            }
-            if ($publishedNode->getNodeType()->isOfType($disabledNodeType)) {
-                $this->systemLogger->log(vsprintf('Redirect skipped based on the current node type (%s) for node %s because is of type %s', [
-                    $publishedNode->getNodeType()->getName(),
-                    $publishedNode->getContextPath(),
-                    $disabledNodeType
-                ]), LOG_DEBUG, null, 'RedirectHandler');
-                return true;
+        if (isset($this->restrictByNodeType)) {
+            foreach ($this->restrictByNodeType as $disabledNodeType => $status) {
+                if ($status !== true) {
+                    continue;
+                }
+                if ($publishedNode->getNodeType()->isOfType($disabledNodeType)) {
+                    $this->systemLogger->log(vsprintf('Redirect skipped based on the current node type (%s) for node %s because is of type %s', [
+                        $publishedNode->getNodeType()->getName(),
+                        $publishedNode->getContextPath(),
+                        $disabledNodeType
+                    ]), LOG_DEBUG, null, 'RedirectHandler');
+                    return true;
+                }
             }
         }
+
         return false;
     }
 
     /**
-     * Check if the current node path is disable by Settings
+     * Check if the current node path is restricted by Settings
      *
      * @param NodeInterface $publishedNode
      * @return bool
      */
-    protected function isPathDisabled(NodeInterface $publishedNode)
+    protected function isRestrictedByPath(NodeInterface $publishedNode)
     {
-        foreach ($this->pathPrefixConfiguration as $pathPrefix => $status) {
-            if ($status !== false) {
-                continue;
-            }
-            $pathPrefix = rtrim($pathPrefix, '/') . '/';
-            if (mb_strpos($publishedNode->getPath(), $pathPrefix) === 0) {
-                $this->systemLogger->log(vsprintf('Redirect skipped based on the current node path (%s) for node %s because prefix matches %s', [
-                    $publishedNode->getPath(),
-                    $publishedNode->getContextPath(),
-                    $pathPrefix
-                ]), LOG_DEBUG, null, 'RedirectHandler');
-                return true;
+        if (isset($this->restrictByPathPrefix)) {
```

Comment:
Prefer an early return here, with a !isset to avoid the identation level

Reply:
alright, I'll change it